.env

```diff
@@ -1 +1 @@
-REACT_APP_BACKEND_BASE_URL=http://localhost:5000
\ No newline at end of file
+REACT_APP_BACKEND_BASE_URL=http://localhost:8000/api/v1
```

Comment:
I didn't see there a port changes to the backend

Reply:
do you mean use .env.local for testing? I'm currently using .env for local testing, 5000 not working for mac, I was thinking to gitignore this .env file, and set up in the production environment variables manually.